sdk/src/main/java/org/coralibre/android/sdk/internal/device_info/DeviceInfo.java

```diff
@@ -0,0 +1,97 @@
+package org.coralibre.android.sdk.internal.device_info;
+
+import static org.coralibre.android.sdk.internal.device_info.ConfidenceLevel.getConfidenceLevel;
+
+public class DeviceInfo implements Comparable<DeviceInfo> {
+
+    private String manufacturer;
+    private String device;
+    private String model;
+    private int rssiCorrection;
+    private int tx;
+    private ConfidenceLevel calibration_confidence;
```

Comment:
Could you rename this to "calibrationConfidence"?

Reply:
done